html_xblock/static/js/html.js

```diff
@@ -0,0 +1,78 @@
+/* Javascript for HTMLXBlock. */
+
+const CUSTOM_FONTS = "Default='Open Sans', Verdana, Arial, Helvetica, sans-serif;";
+const STANDARD_FONTS = "Andale Mono=andale mono,times;" + "Arial=arial,helvetica,sans-serif;" + "Arial Black=arial black,avant garde;" + "Book Antiqua=book antiqua,palatino;" + "Comic Sans MS=comic sans ms,sans-serif;" + "Courier New=courier new,courier;" + "Georgia=georgia,palatino;" + "Helvetica=helvetica;" + "Impact=impact,chicago;" + "Symbol=symbol;" + "Tahoma=tahoma,arial,helvetica,sans-serif;" + "Terminal=terminal,monaco;" + "Times New Roman=times new roman,times;" + "Trebuchet MS=trebuchet ms,geneva;" + "Verdana=verdana,geneva;" + "Webdings=webdings;" + "Wingdings=wingdings,zapf dingbats";
+const FONTS = CUSTOM_FONTS + STANDARD_FONTS;
+
+const PLUGINS = ["codesample", "image", "link", "lists", "textcolor", "codemirror"];
+const PLUGINS_DIR = "/resource/html5/public/plugins/";
+const EXTERNAL_PLUGINS = PLUGINS.map(function(p) { return PLUGINS_DIR + p + "/plugin.min.js" });
+
+function HTML5XBlock(runtime, element) {
+  const editorTab = document.getElementById("editor-tab");
+  const contentSelector = "textarea#html5-textarea";
+  var editor;
+
+  window.addEventListener('load', function () {
```

Comment:
You would need to use [this code](http://youmightnotneedjquery.com/#ready) instead, because `window.addEventListener('load', ...)` doesn't do anything if the document has already loaded, which is the case when opening the editor in Studio.

In fact, since the XBlock runtime only calls the XBlock initialization after the DOM is ready, I think you can just delete this line.